packages/dds/task-manager/src/interfaces.ts

```diff
@@ -19,11 +19,18 @@ export interface ITaskManagerEvents extends ISharedObjectEvents {
 
 export interface ITaskManager extends ISharedObject<ITaskManagerEvents> {
     /**
-     * Try to lock the task.  Promise resolves when the lock is acquired, or rejects if we are removed from the
-     * queue without acquiring the lock for any reason.
+     * Volunteer to lock the task.  Promise resolves true when the lock is acquired, or rejects if we are removed from
```

Comment:
Could you call out reasons we can be rejected here? It will be easier to understand how you can build use cases around it.